src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Select.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+import React from 'react';
+import {action, computed, observable} from 'mobx';
+import {observer} from 'mobx-react';
+import selectStyles from './select.scss';
+import Divider from './Divider';
+import OverlayList from './OverlayList';
+import Option from './Option';
+import Label from './Label';
+
+@observer
+export default class Select extends React.PureComponent {
+    props: {
+        value?: string,
+        onChange?: (s: string) => void,
```

Comment:
It's a really strange behavior. Sometimes it throws an error and sometimes it doesn't. I'll leave it out for now.